src/tools/cephfs/cephfs-shell

```diff
@@ -340,6 +340,7 @@ class CephFSShell(Cmd):
         self.set_prompt()
         self.interactive = False
         self.umask = '2'
+        self.debug = True
```

Comment:
Do you really want to merge this change too? I think it's best to enable debug all the time.

Better thing to do would be, IMHO, to make shell read conf file (so that it can get value of `debug` option from it) and set `debug` option to `True` in the template ceph.conf inside `vstart.sh`. This would make us/devs get traceback all the time withouth hardcoding.

Reply:
This option is to enable debug always. I agree, it is a good idea to extract debug from conf file. I will remove this option from shell.